blocks/cards/cards.css

```diff
@@ -397,13 +445,9 @@ main .nolink .cards > ul > li > .cards-card-body > h6 {
     margin: 0;
   }
 
-  main .block.cards.machine-parts > ul > li .cards-card-image {
-    height: 100%;
-    min-width: 50%;
-  }
 
   main .block.cards.machine-parts > ul > li > div {
-    flex: 1;
+    /*flex: 1;*/
```

Comment:
I think you don't have the commit hook installed, the linting should have caught this, right, @sdmcraft ?